bpf/ep_config.h

```diff
@@ -36,6 +36,9 @@ DEFINE_U32(SECLABEL_NB, 0xfffff);
 DEFINE_U32(POLICY_VERDICT_LOG_FILTER, 0xffff);
 #define POLICY_VERDICT_LOG_FILTER fetch_u32(POLICY_VERDICT_LOG_FILTER)
 
+DEFINE_U32(THIS_INTERFACE_IFINDEX, 0);
```

Comment:
@ti-mo is this still the right way to do constants?

Reply:
Yes, as long as it's consistent with existing code, not really an issue.